dom.bs

```diff
@@ -6035,20 +6035,10 @@ given a <var>document</var>, <var>localName</var>, <var>namespace</var>, and opt
      <li><p>Set <var>result</var> to the result of <a>constructing</a> <var>C</var>, with no
      arguments.
 
-     <li>
-      <p>If <var>result</var> does not implement the {{HTMLElement}} interface, then <a>throw</a> a
-      <code>TypeError</code>.
-
-      <div class=note>
-       <p>This is meant to be a brand check to ensure that the object was allocated by the HTML
-       element constructor. See <a href="https://github.com/heycam/webidl/issues/97">webidl #97</a>
-       about making this more precise.
-
-       <p>If this check passes, then <var>result</var> will already have its
-       <a for=Element>custom element state</a> and <a for=Element>custom element definition</a>
-       initialized.
-      </div>
-     </li>
+     <li><p>Assert: <var>result</var>'s <a for=Element>custom element state</a> and
+     <a for=Element>custom element definition</a> are initialized.
+
+     <li><p>Assert: <var>result</var>'s <a for=Element>namespace</a> is the <a>HTML namespace</a>.
```

Comment:
I think it's still worth keeping a note why this assert is true: it's because IDL enforced that _result_ is a HTMLElement instance, and at this point in time all HTMLElement instances have the HTML namespace.

Reply:
That makes it sound like it might change to me, but I've added a similar note.